.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: ['uphold'],
```

Comment:
Don't we need to add `eslint-config-uphold` to the project?

Reply:
Technically, it `uphold-scripts` should have a peer dependency on our eslint config, and eslint as well. However, it’s currently a normal dependency and we are relying on the fact that the dependencies will be deduped and installed at the root. This is mostly fine.